browser.js

```diff
@@ -1,11 +1,11 @@
 'use strict'
 
-var format = require('quick-format-unescaped')
+const format = require('quick-format-unescaped')
```

Comment:
@davidmarkclements are there browsers we support that do _not_ support `let` and `const`?

Reply:
I would land this and see if anybody complains. IE is finally out of the door, so I would not bother.